insights/config/specs.py

```diff
@@ -437,6 +437,7 @@
     "uname"                     : First([CommandSpec("/bin/uname -a"),
                                     SimpleFileSpec("uname")]),
     "up2date"                   : SimpleFileSpec("etc/sysconfig/rhn/up2date"),
+    "up2date.log"               : SimpleFileSpec("var/log/up2date", large_content=True),
```

Comment:
Better use the same spec name in 1.x and 3.x for the newly added spec.

Reply:
I check the parser for rhsm.log in both 1.x and 3.x:
In 1.x, the spec is "rhsm.log"
In 3.x, the spec is rhsm_log with alias rhsm.log(using Specs.rhsm_log in parser file. If rhsm.log, Specs.rhsm.log seems a little werid).

So use up2date_log in both 1.x and 3.x will be better?

Reply:
use up2date_log instead of up2date.log as same as 3.x